ftl_installer/parser/Parser.py

```diff
@@ -2,6 +2,17 @@
 import argparse
 import ftl_installer
 import logging
+import pprint
+import yaml
+
+class ConfigHandler(object):
+    '''
+    Class to handle the configuration given with `--config` option
+    '''
+
+    def __init__(self, config_file):
+        with open(config_file) as config_file:
+            self.stored_config = yaml.safe_load(config_file)
```

Comment:
Having a single method to generate the config object works for now, but eventually we will want to support reading the config from a known file (if present) as well as generating the config during the installer runtime. The management of which of these to do will be handled by the Parser module (or delegated to a different module).

With that in mind, I would expect there to be (at least) two different ways to create a config object:
- initialize an "empty" config that will be used during runtime when an existing config is not found/provided. I would expect this to be handled by the __init__() method
- create a config based on the existing/provided file. Possibly using a from_file() or from_yaml() method.